anthos-onprem-terraform/avmw_user_cluster_metallb/terraform.tfvars.sample

```diff
@@ -0,0 +1,20 @@
+# [START anthos_onprem_terraform_vmware_user_cluster_metallb_tfvars]
+# -------------------- IGNORE ABOVE THIS LINE -----------------------
+
+project_id          = "YOUR_PROJECT_ID"
+region              = "GCP_REGION_TO_USE"
+admin_cluster_name  = "YOUR_ADMIN_CLUSTER_NAME"
+on_prem_version  = "VERSION_OF_CLUSTER"
+cluster_name        = "avmw-user-cluster-metallb"
+control_plane_node_cpus = 4
+control_plane_node_memory = 8192
+control_plane_node_replicas = 3
+control_plane_vip   = "10.200.0.50"
```

Comment:
Instead of an actual IP address, please add a placeholder: CONTROL_PLANE_VIP.
@Shabirmean has actual IP values in his sample because it was designed to go along with the doc that lets users create ABM clusters on GCE VMs using a script. And in that doc, we know what the IP addresses are, so we can list them in the ABM tfvars file.